tests/client/core/containers/TestHandleLogin.js

```diff
@@ -9,34 +9,15 @@ import { createStore } from 'redux';
 
 import HandleLogin, { mapDispatchToProps } from 'core/containers/HandleLogin';
 import * as api from 'core/api';
-import { userAuthToken } from 'tests/client/helpers';
+import { RouterStub, userAuthToken } from 'tests/client/helpers';
```

Comment:
It would be nice if this used `withRouter()` as well and just passed in the prop but you could leave it if you like. Thanks for extracting this, unfortunately I don't think it should have been done this way.

I'll be sure to give past-Mark a talking to.